Library/Formula/node.rb

```diff
@@ -73,22 +74,28 @@ def install
     system "make", "install"
 
     if build.with? "npm"
-      resource("npm").stage buildpath/"npm_install"
-
+      npm_buildpath = buildpath/"npm_install"
+      resource("npm").stage npm_buildpath
```

Comment:
I'm not sure there's a huge amount of benefit in splitting this onto two lines, purely IMO.